### PR TITLE
Make additional changes for Schools Admissions

### PR DESCRIPTION
**Pull request by bella-cockrell**
# Link to JIRA ticket

https://hackney.atlassian.net/browse/DOC-809

# Describe this PR
## What is the problem we're trying to solve
School's Admissions has requested that we make a change to the search feature. They would like to filter on evidence requests that contain documents that need to be approved.

# What changes have we introduced

We've added in changes to the FE that call the API for evidence requests based on the state of the request (e.g. PENDING, IN REVIEW, etc). 

# Checklist

- [x]  Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x]  Added tests to cover all new production code
- [ ]  Added comments to the README or updated relevant documentation (add link to documentation), where necessary.
- [x]  Checked all code for possible refactoring
- [x]  Code pipeline builds correctly

### CODE REVIEW

**Review by bogdan-zaharia — APPROVED** [review]
LGTM

File: src/boundary/response-mapper.ts [hunk 1]
```diff
@@ -80,6 +80,12 @@ export class ResponseMapper {
     return attrs.map((r) => this.mapResidentResponse(r));
   }
 
+  static mapEvidenceResponseList(
```
[the remark this answers is not in the document]
Comment (bella-cockrell) [reply src/boundary/response-mapper.ts:83]: No, I can remove!

File: src/gateways/internal-api.test.ts [hunk 2]
```diff
@@ -274,4 +281,73 @@ describe('Internal API Gateway', () => {
...
+    const team = 'Development Team';
+
+    describe('when successful', () => {
+      beforeEach(() => {
+        client.get.mockClear();
+        client.get.mockResolvedValue({
+          data: apiResponse,
+        });
+
+        apiResponse.map((_, i) =>
```
[the remark this answers is not in the document]
Comment (bella-cockrell) [reply src/gateways/internal-api.test.ts:297]: Good spot! Have taken that in

**Comment by bella-cockrell** [comment]
Andy's changes can be found in this PR: https://github.com/LBHackney-IT/document-evidence-store-frontend/pull/221

**Pull request merged**